keyboards/playkbtw/pk64rgb/readme.md

```diff
@@ -0,0 +1,17 @@
+# Play Keyboard 64RGB
+
+![PK64RGB](https://cdn.store-assets.com/s/409567/i/25281841.png)
+
+A 60% RGB hotswap keyboard PCB made by Play Keyboard.
+
+* Keyboard Maintainer: [Barry Huang](https://github.com/yj7272098)
+* Hardware Supported: Play Keyboard 64 RGB
+* Hardware Availability: [Play Keyboard](http://play-keyboard.store/)
+
+Make example for this keyboard (after setting up your build environment):
```

Comment:
Could you add instructions on how to enter the bootloader, to flash new firmware? 

Eg: 
```markup
## Bootloader

Enter the bootloader in 3 ways:

* **Bootmagic reset**: Hold down the key at (0,0) in the matrix (usually the top left key or Escape) and plug in the keyboard
* **Physical reset button**: Briefly press the button on the back of the PCB - some may have pads you must short instead
* **Keycode in layout**: Press the key mapped to `RESET` if it is available
```